keyboards/wilba_tech/rama_works_m60_a/keymaps/zyber/keymap.c

```diff
@@ -0,0 +1,41 @@
+// M60-A layout
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```